src/components/Home.js

```diff
@@ -0,0 +1,52 @@
+import { useSelector } from 'react-redux';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Header from './Header';
+import Banner from '../image/Banner.png';
+
+const HomePage = () => {
+  const [search, setSearch] = useState('');
+  const northAmericaCountries = useSelector((state) => state.continentReducer);
+  const filteredCountries = northAmericaCountries.filter((country) => {
+    const newSearch = country.country.toLowerCase().includes(search);
+    return newSearch;
+  });
+  return (
```

Comment:
- [ ] I kindly suggest that you ensure that your Homepage has a filter e.g search bar or dropdown list to filter the list of items displayed in your homepage. For your project, you could add a filter by continent.